app/controllers/admin/feedback_messages_controller.rb

```diff
@@ -3,7 +3,7 @@ class FeedbackMessagesController < Admin::ApplicationController
     layout "admin"
 
     def index
-      @q = FeedbackMessage.includes(:reporter, :offender, :affected)
+      @q = FeedbackMessage.includes(:offender, :affected)
```

Comment:
I mentioned this in a comment previously, but I think the system wants me to formally request changes, so... I think instead we should add `reporter` to the bullet configuration [here](https://github.com/forem/forem/blob/a6030022f6bae2d75d021a53bb2684ec25989380/config/environments/development.rb#L135-L145) as (another) `unused_eager_loading`. 

It should be fairly straight-forward, but let me know if you want to pair on it, sometimes nice to have an extra set of eyes!

Reply:
I have applied this change. Thanks @jaw6 for clarification.

`Bullet.add_safelist(type: :unused_eager_loading, class_name: "FeedbackMessage", association: :reporter)`